content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,142 @@
+- If project is created after General Availability, it starts on the `Free` plan.
+
+## Comparison of plans
+
+For plan pricing details, go to our [Pricing](https://cloud.prisma.io/pricing) page.
+
+|                                               | `Early Adopter`                                                  | `Free`                                                   | `Scale`                                                                                                                                                                   | `Enterprise`                                 |
+| --------------------------------------------- | ---------------------------------------------------------------- | -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| Team members                                  | 10                                                               | 3                                                        | Unlimited<br /><br /> <strong>Quota</strong><br /><br />5 <br /><br /><strong>Overage</strong><br /><br />Each additional member at a fixed price                         | Custom                                       |
+| Member roles                                  | • Admin<br />• Viewer<br />• Developer<br />• Collaborator       | Admin only                                               | • Admin<br />• Viewer<br />• Developer<br />• Collaborator                                                                                                                | • Admin<br />• Viewer<br />• Developer<br /> |
+| Environments                                  | 7                                                                | 2                                                        | Unlimited<br /><br /> <strong>Quota</strong><br /><br />5 <br /><br /><strong>Overage</strong><br /><br />Each additional member at a fixed price                         | Custom                                       |
+| **Data Proxy**<br /><br /> - CPU usage        | 100,000,000<br /><br />CPU&nbsp;ms<br /><br />(hard limit)       | 3,000,000<br /><br />CPU&nbsp;ms<br /><br />(hard limit) | Unlimited<br /><br /><strong>Quota</strong><br /><br />30,000,000 CPU&nbsp;ms<br /><br /><strong>Overage</strong><br /><br />Each additional CPU&nbsp;ms at a fixed price | Unlimited<br /><br />Custom pricing          |
+| **Data Proxy**<br /><br /> -&nbsp;Concurrency | • 3 QEIDs\* per project<br />• 5 QEIDs\* per account/team member | 100 QEIDs\* per project                                  | 100 QEIDs\* per project                                                                                                                                                   | Custom                                       |
```

Comment:
>5 QEIDs* per account/team member

This was lifted and no longer applies, since it would also kick in when using Free or Scale projects in the same account.

Reply:
Ok, I've removed this.